k8s/tf/200_single_state/eu-central-1/eks.tf

```diff
@@ -1,6 +1,7 @@
 locals {
   cluster_features = {
-    "aws_calico"         = true
+    # "aws_calico"         = true # added using helm, ref: https://docs.aws.amazon.com/eks/latest/userguide/calico.html
+    # "aws-load-balancer-controller" # added using eks and helm, ref: https://docs.aws.amazon.com/eks/latest/userguide/aws-load-balancer-controller.html
```

Comment:
Why is this line commented out? Is there an `= true` missing?

Reply:
I removed calico installed from the EKS module and installed a most recent version using helm following the doc wondering it could help on requests hanging, it didn't change at the end, that's why I just updated in eu-central-1, and keep in us-west-2 as is.

Reply:
I just put it as a reference, there is no parameter "aws-load-balancer-controller", I can move it to the top of the file to avoid confusion.